modules/Classroom/components/StudentSearchModal/StudentSearchModal.tsx

```diff
@@ -92,17 +92,19 @@ const StudentSearchModal: React.FC<StudentSearchModalProps> = ({
               Loading students...
             </Text>
           )}
-          {isEnrolling && (
-            <Text color="dimmed" size="sm" mt="sm">
-              Enrolling student...
-            </Text>
-          )}
           <Group mt="xl" mb="sm" position="right">
             <Button size="sm" onClick={handleClose} style={{ background: "#4CAF50" }}>
               Cancel
             </Button>
-            <Button type="submit" size="sm" color="green" style={{ background: "#4CAF50" }}>
-              Enroll
+            <Button
+              type="submit"
+              size="sm"
+              color="green"
+              style={{ background: "#4CAF50" }}
+              disabled={isEnrolling}
+              loading={isEnrolling}
+            >
+              {!isEnrolling && "Enroll"}
```

Comment:
Do you need this check?

Reply:
Bhaiya, I don't want to show the "Enroll" word when it's clicked, it will show loading animation only. You can see this on the video I provided.

Reply:
Wouldn't just adding loading to be true achieve that?

Reply:
Yes the loading animation was showing but "Enroll"  word was also there.